lib/node_modules/@stdlib/math/base/complex/divide/test/test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var cdiv = require( './../lib' );
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof cdiv, 'function', 'main export is a function' );
+	t.end();
+});
+
+function testCdiv( a, b, c, d, e, f, label ) {
```

Comment:
That makes sense, but is there a way to test lots of explicit cases without hundreds of lines of code? In this case I needed very specific tests to test all branches of the code, which could add up to lots of lines of code very quickly if they're all written out. The main goal was just that fewer lines = less maintenance.